VERSIONS.md

```diff
@@ -2,7 +2,7 @@
 
 | Gem    | jQuery UI |
 |--------|-----------|
-| 7.0.1  | 1.13.2    |
+| 7.0.2  | 1.13.2    |
```

Comment:
Should you maybe keep both?

Reply:
Nope, I kind of accidentally skipped 7.0.1 and only published 7.0.2 : https://rubygems.org/gems/jquery-ui-rails-dox-fork